src/components/HomePage/HomePage.vue

```diff
@@ -0,0 +1,47 @@
+<script lang="ts" setup>
+import { useRouter } from "vue-router";
+const routerManager = useRouter();
+
+interface ButtonSchema {
+  name: string;
+  label: string,
+  color: string
+  to: string
+}
+
+const buttons: ButtonSchema[] = [
+  {
+    name: 'Register User',
+    label: 'Register User',
+    color: 'red',
+    to: 'Form'
+  },
+  {
+    name: 'About',
+    label: 'List Users',
+    color: 'red',
+    to: 'About'
+  },
+];
```

Comment:
You can just define two buttons within the template, all of your button share the same color so you can also remove the `style` property and add the color to the style.
```suggestion
```

Reply:
done